controllers/crypto.js

```diff
@@ -0,0 +1,26 @@
+const { fetchProducts } = require('../models/crypto')
+
+const ERROR_MESSAGE = 'Something went wrong. Please try again or contact admin'
+
+/**
+ * Get the  products details
+ * @param {Object} req - Express request object
+ * @param {Object} res - Express response object
+ */
+
+const getProducts = async (req, res) => {
```

Comment:
Can we name the controller file as `cryptoController.js`, similar to other files?